maker/src/routes.rs

```diff
@@ -1,56 +1,46 @@
+use crate::bitmex::Quote;
+use anyhow::Result;
 use http_api_problem::HttpApiProblem;
 use http_api_problem::StatusCode;
 use rocket::serde::json::Json;
 use rocket::serde::Deserialize;
 use rocket::serde::Serialize;
+use rocket::State;
+use rust_decimal::Decimal;
+use rust_decimal_macros::dec;
 use ten_ten_one::wallet::force_close_channel;
-
-#[derive(Serialize, Deserialize, Debug)]
-pub struct Price(u64);
-
-impl From<u64> for Price {
-    fn from(val: u64) -> Self {
-        Self(val)
-    }
-}
+use tokio::sync::watch;
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct Offer {
-    bid: Price,
-    ask: Price,
-}
-
-#[derive(Serialize, Deserialize, Debug)]
-pub struct Offers {
-    long: Offer,
-    short: Offer,
-    index_price: Price,
+    bid: Decimal,
+    ask: Decimal,
+    index: Decimal,
```

Comment:
Does this work or do we need to annotate it with how to serialize like you did below?

Reply:
it works as it is serializing the decimal to a string and then i am parsing it from string into a float. It's probably nicer to serialize it as float. good catch - i am going to adapt.